Azure-Samples/communication-services-web-calling-tutorial
# Mute / Unmute Incoming Audio

## Purpose

Added new speaker icon next to the mute icon to demonstrate the mute/unmute incoming audio feature. 

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
Join call.
When click on the mute speaker icon then the incoming audio should be muted (not call audio will be played).
When click on the unmute speaker icon then the incoming audio should start playing.
```

## What to Check
Verify that the following are valid
* ...

## Other Information
